src/App.jsx

```diff
@@ -1,28 +1,47 @@
 import { createBrowserRouter, RouterProvider } from 'react-router-dom';
 
-import Header from './components/Header';
 import Home from './containers/Home';
+import About from './containers/About';
 import ArtWorkDetail, { loader as artWorkLoader } from './containers/ArtWorkDetail';
-import Footer from './components/Footer';
+import NotFound from './containers/NotFound';
+import Layout from './components/Layout';
 
 const router = createBrowserRouter([
   {
     path: '/',
-    element: <Home />,
+    element: (
+      <Layout>
+        <Home />
+      </Layout>
+    ),
   },
   {
     path: '/art-works/:artWorkId',
-    element: <ArtWorkDetail />,
+    element: (
+      <Layout>
+        <ArtWorkDetail />
+      </Layout>
+    ),
     loader: artWorkLoader,
   },
+  {
+    path: '/about/',
+    element: (
+      <Layout>
+        <About />
+      </Layout>
+    ),
+  },
+  {
+    path: '*',
```

Comment:
This is a good attempt at representing a not-found page. However, this is not covering some edge cases, like when you navigate to the page http://localhost:3000/art-works/invalid-url this is showing an application error instead of the actual not found component.

Reply:
Sorry for the confusion. The wildcard `'*'` is fine, you just need to handle this error from the `ArtworkDetail` component. One way of resolving it is performing a redirect programmatically to the `NotFound` component when the art work isn't found.